cmd/fleetctl/scripts.go

```diff
@@ -75,8 +74,6 @@ func runScriptCommand() *cli.Command {
 				return err
 			}
 
-			fmt.Println("\nScript is running. Please wait for it to finish...")
```

Comment:
Have we checked with product team if we are ok with removing this for the case when scripts are enabled? (Maybe we can amend the message to be something like `\nAttempting to run script. Please wait for it to finish...`)

Reply:
Hmm I think this may have been included by accident when I was fixing some merge conflicts. Taking a look.

Reply:
This was probably just a mistake on my part. Reverted.